langlib/langlib-test/src/test/resources/test-src/valuelib_test.bal

```diff
@@ -762,9 +787,10 @@ function testFromJsonWithTypeXML() {
     string s1 = "<test>name</test>";
     xml|error x1 = s1.fromJsonWithType(xml);
     assert(x1 is xml, true);
-    xml x11 = <xml> x1;
-    json|error j = x11.toJson();
-    assert(<json> j, s1);
+    if (x1 is xml) {
```

Comment:
Again we are silently ignoring the error. I think we should do

```ballerina
    xml x11 = checkpanic x1;
    json|error j = x11.toJson();
    assert(<json> j, s1);
```

Reply:
Fixed